tests/test_heatserts.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import math
+import importlib
+import itertools
+import cadquery as cq
+import plugins.heatserts.heatserts as heatsert_module
+import numbers
+
+
+@pytest.fixture
+def box():
+    return cq.Workplane().box(20, 20, 20)
+
+
+possible_chamfer_vals = [0.1, 0.5, 2]
+
+
+@pytest.mark.parametrize("size", heatsert_module.heatsert_dims.keys())
+class TestSizes:
+    def test_heatsert_makes_a_hole(self, box, size):
+
+        box_with_hole = box.faces(">Z").workplane().heatsert(size)
+        assert box.faces().size() < box_with_hole.faces().size()
+
+    def test_heatsert_basic_dims(self, box, size):
+
+        box_volume = box.val().Volume()
+
+        dims = heatsert_module.heatsert_dims[size]
+        box_with_hole = box.faces(">X").workplane().heatsert(size)
+        volume_cut_out = box_volume - box_with_hole.val().Volume()
+        heatsert_volume = math.pi * (dims.diam / 2) ** 2 * dims.depth
+        assert volume_cut_out > heatsert_volume - 0.1
+
+    def test_heatsert_bolt_clear(self, box, size):
+
+        _, depth, bolt_diam = heatsert_module.heatsert_dims[size]
+        bolt_clear = 15
+
+        short_hole = box.faces(">Y").workplane().heatsert(size)
+        long_hole = box.faces(">Y").workplane().heatsert(size, bolt_clear=bolt_clear)
+
+        extra_hole_volume = math.pi * (bolt_diam * 1.2 / 2) ** 2 * (bolt_clear - depth)
+        assert (
+            short_hole.val().Volume() - long_hole.val().Volume()
+            > extra_hole_volume - 0.1
+        )
+
+    @pytest.mark.parametrize(
+        "chamfer_arg",
+        possible_chamfer_vals
+        + [el for el in itertools.product(possible_chamfer_vals, repeat=2)],
+    )
+    def test_heatsert_chamfer(self, box, size, chamfer_arg):
+
+        dims = heatsert_module.heatsert_dims[size]
+
+        face_center = box.faces("<Z").val().Center()
+        chamfered = box.faces("<Z").workplane().heatsert(size, chamfer=chamfer_arg)
+        circles = chamfered.edges("%CIRCLE").vals()
+        radii = [c.radius() for c in circles]
+        print(radii)
```

Comment:
Not sure if you meant to leave this in the test.

Reply:
Nope, thanks.